torch/_inductor/codecache.py

```diff
@@ -1302,18 +1302,7 @@ class VecISA:
 #include <ATen/cpu/vec/vec.h>
 #endif
 
-#ifdef __APPLE__
-// Fix Mac OS UT failed.
-__attribute__((aligned(64))) float in_out_ptr0[16] = {0.0};
-#else
-#if defined(_WIN32)
-#define __at_align__ __declspec(align(64))
-#else
-#define __at_align__ __attribute__((aligned(64)))
-#endif
-
-__at_align__ float in_out_ptr0[16] = {0.0};
-#endif
+alignas(64) float in_out_ptr0[16] = {0.0};
```

Comment:
@xuhancn Does it work on Windows?

Reply:
@jgong5 Is there a  CI to test it on Windows?

Reply:
@xuhancn told me `alignas` is not always supported. @xuhancn can you confirm?

Reply:
> @xuhancn told me `alignas` is not always supported. @xuhancn can you confirm?

If if not support Windows, I will fix it later.